src/OpenTelemetry.Exporter.Prometheus/README.md

```diff
@@ -1,21 +1,124 @@
-# Prometheus Exporter for OpenTelemetry .NET
+# Application monitoring with OpenTelemetry .NET PrometheusExporter via Prometheus and Grafana
```

Comment:
the current title is accurate..

Reply:
I've decoupled the getting started with Prometheus and Grafana from this document.
Please review here: https://github.com/open-telemetry/opentelemetry-dotnet/pull/2795.
Thank you.